templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml

```diff
@@ -131,6 +131,7 @@ Resources:
             reason: No KMS Key Provided
     Condition: cCreateCloudWatchLogGroup
     Properties:
+      KmsKeyId: !Ref AWS::NoValue
```

Comment:
why do we have no value for kms key id ? Is it kmskey id automatically generated by AWS?

Reply:
LogGroup don't need KMS